src/hooks/UrqlUseMutation.re

```diff
@@ -38,9 +43,18 @@ let useMutationResponseToRecord = (result: useMutationResponseJs('a)) => {
   {fetching, data, error, response};
 };
 
-let useMutation = (~query: string) => {
-  let (useMutationResponseJs, executeMutation) = useMutationJs(~query);
+let useMutation =
+    (
+      ~request: {
+         .
+         "query": string,
+         "variables": Js.Json.t,
+         "parse": Js.Json.t => 'response,
+       },
+    ) => {
+  let (useMutationResponseJs, executeMutation) =
+    useMutationJs(~query=request##query);
```

Comment:
Since we only have a single parameter for `useMutationJs`, do you mind switching that `external` to _not_ use a labelled argument for `query`? I know I had it that way originally 😅, but I don't see much use in it now.